app/src/main/java/ch/epfl/cs311/wanderwave/ui/screens/LoginScreen.kt

```diff
@@ -54,7 +57,12 @@ fun LoginScreen(
 
   Column(modifier = Modifier.testTag("loginScreen")) {
     LoginAppLogo(modifier = Modifier.weight(1f))
-    WelcomeTitle(modifier = Modifier.weight(4f))
+    AppLogoWithTitle(
+        modifier = Modifier.weight(4f),
+        initialColor = MaterialTheme.colorScheme.primary,
+        finalColor = Color.Red,
```

Comment:
The red should be in the theme, because we should keep the app color scheme consistent.

Reply:
I think its not really part of the theme but rather a fixed color in this case